pd/synth/cva6_read.tcl

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright 2021 Thales DIS design services SAS
+#
+# Licensed under the Solderpad Hardware Licence, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.0
+# You may obtain a copy of the License at https://solderpad.org/licenses/
+#
+# Original Author: Jean-Roch COULON (jean-roch.coulon@thalesgroup.com)
+#
+
+source -echo -verbose scripts/dc_setup.tcl
+
+#read_ddc ${DCRM_FINAL_DDC_OUTPUT_FILE}
```

Comment:
```suggestion
```

Reply:
Sorry, the comment was not right. The goal is to read the final database. I have removed the comment.